state-chain/chains/src/dot.rs

```diff
@@ -243,7 +243,9 @@ pub struct PolkadotTrackedData {
 impl Default for PolkadotTrackedData {
 	#[track_caller]
 	fn default() -> Self {
-		panic!("You should not use the default chain tracking, as it's meaningless.")
+		frame_support::print("You should not use the default chain tracking, as it's meaningless.");
```

Comment:
👍  yeah was looking to move these away from a panic, think it makes more sense like this + is safer